AUTHENTICATION.md

```diff
@@ -28,22 +22,24 @@ providing **Project ID** and **Service Account Credentials** directly in code.
 
 ### Google Cloud Platform environments
 
-While running on Google Cloud Platform environments such as Google Compute Engine, Google App Engine and
-Google Kubernetes Engine, no extra work is needed. The **Project ID** and **Credentials** and are 
-discovered automatically from the attached service account. Code should be written as if already authenticated.
+While running on Google Cloud Platform environments such as Google Compute Engine, Google App Engine
+and Google Kubernetes Engine, no extra work is needed. The **Credentials** and are discovered
+automatically from the attached service account. Code should be written as if already authenticated.
 
 For more information, see
 [Set up ADC for Google Cloud services](https://cloud.google.com/docs/authentication/provide-credentials-adc#attached-sa).
 
 ### Environment Variables
 
-**NOTE**: This library uses [`getenv`](https://www.php.net/manual/en/function.getenv.php), so if your environemnt 
-variables are set in PHP, they must use [`putenv`](https://www.php.net/manual/en/function.putenv.php),
+**NOTE**: This library uses [`getenv`](https://www.php.net/manual/en/function.getenv.php), so if
+your environemnt variables are set in PHP, they must use
+[`putenv`](https://www.php.net/manual/en/function.putenv.php),
 
 ```php
 putenv("GOOGLE_APPLICATION_CREDENTIALS=" . __DIR__ . '/your-credentials-file.json');
 ```
-The **Project ID** and **Credentials JSON** can be placed in environment variables instead of declaring them directly in code.
+The **Credentials JSON** can be placed in environment variables instead of
+declaring them directly in code.
 
 Here are the environment variables that Google Cloud PHP checks for project ID:
```

Comment:
This section (and the two environment variables below it) should be removed, since they only apply to some Handwritten libraries, and not to GAPIC. 

Or we can keep them, but move them into its own section at the very bottom of this page labeled "Project ID Detection", where we explicitly list the _specific services_ which support it. I think this is the best option, but it will require slightly more work.

Packages which support project ID detection

 - Bigtable
 - PubSub
 - Storage
 - Spanner
 - BigQuery
 - Datastore
 - Firestore
 - Debugger
 - Logging
 - Trace
 - Translate